First stab at adding Hebrew support.

clsn opened the pull request:
So I thought I'd try to add Hebrew characters.  Should be doable within the structure.  Actually came out surprisingly well for a first try.  
Some of the letters maybe need redesigning; particularly מ and probably ל and maybe ק; smaller improvements maybe to a few others.  I didn't "remove overlap" most of the time, but left things as overlapping pieces so we can more easily adjust them if they're off a little.  It's just a first attempt; let me know if it's in the right direction at least.

Commit:
First stab at adding Hebrew support.

Some of the letters maybe need redesigning, but many are good.
includes all letters, vowels, and precomposed chars.

rbanffy commented:
Wow! Thanks. Let me look into it.

On Tue, Jun 10, 2025 at 9:59 PM Mark Shoulson ***@***.***> wrote:
>
> So I thought I'd try to add Hebrew characters. Should be doable within the structure. Actually came out surprisingly well for a first try.
> Some of the letters maybe need redesigning; particularly מ and probably ל and maybe ק; smaller improvements maybe to a few others. I didn't "remove overlap" most of the time, but left things as overlapping pieces so we can more easily adjust them if they're off a little. It's just a first attempt; let me know if it's in the right direction at least.
>
> ________________________________
>
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/rbanffy/3270font/pull/139
>
> Commit Summary
>
> bebda73 First stab at adding Hebrew support.
>
> File Changes (1 file)
>
> M 3270_HQ.sfd (7308)
>
> Patch Links:
>
> https://github.com/rbanffy/3270font/pull/139.patch
> https://github.com/rbanffy/3270font/pull/139.diff
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>



-- 
Ricardo Bánffy
http://about.me/rbanffy

rbanffy commented:
If possible, can you export the symbols to a separate SFD file, checkout the 3270_HQ.sfd file so it's unchanged, and change the PR to be against the `develop` branch? This way I won't miss any symbols you changed, and I can add them to the develop branch with proper credit.

Commit:
Tweaked the shape of מ.

Commit:
Revert 3270_HQ.sfd to main; add 3270_HQ+hebrew.sfd

clsn commented:
I might have only confused things more, I'm afraid, but I've added the 3270_HQ+hebrew.sfd file which is my changed file and reverted the 3270_HQ.sfd file.  I'll resubmit this PR against develop.
